code/game/atoms.dm

```diff
@@ -115,6 +115,14 @@
 
 	///The color this atom will be if we choose to draw it on the minimap
 	var/minimap_color = MINIMAP_SOLID
+	///Icon state of debris when impacted by a projectile
+	var/debris = FALSE
+	///Amount of debris particles
+	var/debris_amount = 8
+	///Velocity of debris particles
+	var/debris_velocity = -15
+	///If the debris particles have a bloom filter
+	var/debris_bloom = FALSE
```

Comment:
Gross here is
A) Atom vars, could this be an element or something instead
B) FALSE/string mixing
C) string should be defines